### ISSUE TITLE
cell format of duration

### ISSUE THREAD

**Issue by ventoux84** [prometheus42/libreoffice-ical-importer#3]
Please, change format from text to hour

**Comment by prometheus42** [comment]
Could you be more specific? Where exactly should the format be `hour` instead of `text`? Which field are you referring to?

**Comment by ventoux84** [comment]
It's about the cell "duration". I use it in Calc and i may do in another cell "=CNUM(duration)"

**Comment by prometheus42** [comment]
It seems to work correctly for me. If I import a sample file into LibreOffice Calc, all cells in the column "duration" are formatted as time value. The cells contain values like 0:02:45 for example.

Isn't that correct?

**Comment by ventoux84** [comment]
Yes, this is the content. But the format doesn't allow to make a calculation. For example, it is not possible to add the duration.
To do that I have to take the CNUM() value of the cell.
But it is really not a big deal.
